hrms/overrides/company.py

```diff
@@ -132,3 +132,79 @@ def validate_default_accounts(doc, method=None):
 					"The currency of {0} should be same as the company's default currency. Please select another account."
 				).format(frappe.bold(_("Default Payroll Payable Account")))
 			)
+
+
+def get_company_data_to_be_ignored():
+	if "erpnext" in frappe.get_installed_apps():
```

Comment:
erpnext will always be there in installed apps because hrms is dependent on erpnext. So this condition is unnecessary